Modules/Tokens/src/Tokens.Jobs.SanityCheck/Infrastructure/DataSource/DataSource.cs

```diff
@@ -9,14 +9,14 @@ namespace Tokens.Jobs.SanityCheck.Infrastructure.DataSource;
 public class DataSource : IDataSource
 {
     private readonly IBlobStorage _blobStorage;
-    private readonly TokenRepositoryOptions _repositoryOptions;
-    private readonly ITokenRepository _tokenRepository;
+    private readonly TokensRepositoryOptions _repositoryOptions;
+    private readonly ITokensRepository _tokensRepository;
 
-    public DataSource(IBlobStorage blobStorage, IOptions<TokenRepositoryOptions> repositoryOptions, ITokenRepository tokenRepository)
+    public DataSource(IBlobStorage blobStorage, IOptions<TokensRepositoryOptions> repositoryOptions, ITokensRepository tokensRepository)
```

Comment:
Remember that we directly used the DbContext instead of the repository in the Messages sanity check